subsystem/feature_support_levels_test.go

```diff
@@ -57,6 +72,34 @@ var _ = Describe("Feature support levels API", func() {
 			return infraEnv, errRegisterInfraEnv
 		}
 
+		Context("GetSupportedFeatures", func() {
+
+			It("With Platform", func() {
```

Comment:
can you add another test for platform + arch? for example: vsphere + ARM or something that is not supported

Reply:
We test those scenarios on the unittests. In the newly added sybsystem tests, we are testing the filtering and not each filter `support-level` value